app/models/recipe_ingredient.rb

```diff
@@ -1,3 +1,9 @@
 class RecipeIngredient < ApplicationRecord
+  belongs_to :recipe
+  belongs_to :ingredient
+
+  def self.find_ingredients(query_id)
+    where("recipe_id = #{query_id}")
```

Comment:
This is a little bit of a code smell, though not super offensively. This reads like you're trying to find all ingredients that belong to a specific recipe, and that's exactly what associations do per instance. But if this is a class method, maybe there are other uses...? I'm not seeing them in this IC though.

Reply:
By "associations per instance", I mean something like
```
@recipe.ingredients
```